required/latex-lab/testfiles-minipage/minipage-002-todo.luatex.tlg

```diff
@@ -166,8 +166,6 @@ Package tagpdf Info: Parent-Child 'P' --> 'MC'.
 (tagpdf)             Relation is 1 (='0..n') on line ...
 tagpdf DEBUG Info: Tagging suspended
 (tagpdf DEBUG)     level: 0 ==> 1, label: \tikzpicture  [on line ...]
-tagpdf DEBUG Info: Struct 12 begin ignored with options: tag=\l__tag_para_main_tag_tl ,attribute-class=\l__tag_para_main_attr_class_tl , 
-(tagpdf DEBUG)     [on line ...]
```

Comment:
why are those no longer showing?

Reply:
because the \tagstruct-commands are now hidden in tagging sockets and so no longer executed and so they no longer can give this message.